core/app/c/[communitySlug]/forms/[formSlug]/edit/actions.ts

```diff
@@ -0,0 +1,30 @@
+"use server";
+
+import { db } from "~/kysely/database";
+import { isUniqueConstraintError } from "~/kysely/errors";
+import { getLoginData } from "~/lib/auth/loginData";
+import { logError } from "~/lib/logging";
+import { autoRevalidate } from "~/lib/server/cache/autoRevalidate";
+import { defineServerAction } from "~/lib/server/defineServerAction";
+
+export const updateForm = defineServerAction(async function updateForm({ formId, name }) {
```

Comment:
this updateForm parameter should be explicitly typed to take a `FormsId` and string, otherwise it's `{formId: any, name: any}`